state/address.go

```diff
@@ -75,20 +75,6 @@ func appendPort(addrs []string, port int) []string {
 	return newAddrs
 }
 
-// Addresses returns the list of cloud-internal addresses that
-// can be used to connect to the state.
-func (st *State) Addresses() ([]string, error) {
-	addrs, err := st.controllerAddresses()
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	config, err := st.ControllerConfig()
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	return appendPort(addrs, config.StatePort()), nil
```

Comment:
Notice this will never be empty. This is important because the uniter tests specifically test for this.